addons/api/addon/adapters/host-catalog.js

```diff
@@ -20,8 +20,12 @@ export default class HostCatalogAdapter extends ApplicationAdapter {
   buildURL(modelName, id, snapshot, requestType) {
     let url = super.buildURL(...arguments);
     if (requestType === 'createRecord') {
-      const pluginName = snapshot.attr('plugin')?.name;
-      if (pluginName) url = `${url}?plugin_name=${pluginName}`;
+      try {
+        const pluginName = snapshot.attr('plugin')?.name;
+        if (pluginName) url = `${url}?plugin_name=${pluginName}`;
+      } catch (e) {
+        // Ignore any adapter errors here
```

Comment:
These try catches are needed now because for some reason, if there's a 404 you get a cache error by just trying to access the snapshot attribute